docs/contents/components/forms/hue-slider/accessibility.ja.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: HueSlider
+description: "`HueSlider`は、ユーザーが色相を選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/color-picker"
+order: 4
+with_description: true
+tab: アクセシビリティ
+---
+
+## キーボード操作
+
+| キー         | 説明                                            | 状態 |
+| ------------ | ----------------------------------------------- | ---- |
+| `ArrowDown`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowLeft`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowRight` | スライダーの値を1ステップ分増加します。         | -    |
+| `ArrowUp`    | スライダーの値を1ステップ分増加します。         | -    |
+| `End`        | スライダーの最大値に移動します。                | -    |
+| `Home`       | スライダーの最小値に移動します。                | -    |
+| `PageDown`   | スライダーの値を大きく減少(全体の10%分)します。 | -    |
+| `PageUp`     | スライダーの値を大きく増加(全体の10%分)します。 | -    |
```

Comment:
Please unify the wording of this section with https://github.com/yamada-ui/yamada-ui/pull/3322 too.

Reply:
fixed.
https://github.com/yamada-ui/yamada-ui/pull/3337/commits/651e4d1957f2037f6df13634a7ce1ddab3beae60